.github/workflows/test.yaml

```diff
@@ -50,3 +50,28 @@ jobs:
         run: npm run format-check --prefix assets
       - name: Run assets tests
         run: npm test --prefix assets
+  windows:
+    runs-on: windows-latest
+    env:
+      MIX_ENV: test
+    steps:
+      - name: Configure Git
+        run: git config --global core.autocrlf input
+        working-directory: .
+      - uses: actions/checkout@v2
+      - name: Install Erlang & Elixir
+        uses: erlef/setup-beam@v1
+        with:
+          otp-version: '24.0'
+          elixir-version: '1.13.0'
+      - name: Start epmd
+        run: cmd /c "START /b epmd"
+        working-directory: ${{ env.INSTALL_DIR_FOR_OTP }}/erts-12.0.4/bin
+      - name: Install mix dependencies
+        run: mix deps.get
+      - name: Check formatting
+        run: mix format --check-formatted
+      - name: Check warnings
+        run: mix compile --warnings-as-errors
+      - name: Run tests
+        run: elixir --cookie "COOKIEFORTESTS" -S mix test
```

Comment:
Just testing

```suggestion
        run: mix test
```

Reply:
Now it failed, so maybe it's just that `elixir` doesn't propagate the exit status. @josevalim you added the explicit cookie as part of #199, is there a specific reason why we do this?

Reply:
Ah, I think it was to assert that using a cookie other than default works. But we now set a random cookie for every run

https://github.com/livebook-dev/livebook/blob/6b7825871338af0ec3f4196ec3e17d2670e6a92c/lib/livebook.ex#L61-L65

so I don't think we need do to this anymore?